### ISSUE TITLE
LinksysVelopConfigFlow calls async_show_progress without passing a progress task, this is not valid and will break in Home Assistant Core 2024.8

### ISSUE THREAD

**Issue by iSteve-O** [uvjim/linksys_velop#453]
Sorry if you know about this already, but I just noticed it in the log and also noticed nobody has reported it yet.

I am using the latest HAOS v2024.4.1 on raspberry pi 5.

```yaml
WARNING (MainThread) [homeassistant.data_entry_flow] custom_components.linksys_velop.config_flow::LinksysVelopConfigFlow calls async_show_progress without passing a progress task, this is not valid and will break in Home Assistant Core 2024.8. Please report it to the custom integration author
```

**Comment by uvjim** [comment]
No need to apologise. Yeah, I'm aware of this one (I have a couple of other integrations that I need to address it in too).

It's on my list of things to get to when life stops getting in the way.

There's a couple of betas queued up for this one that I need to release and then I'll try and get on it.

**Comment by uvjim** [comment]
Should be fixed by #471. Please re-open if it isn't.
